<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-P11DB2I" failures="1" tests="2" name="Surefire test" time="34.09" errors="0" timestamp="2023-10-08T16:00:49 IST">
  <testcase classname="AutomationAnywhereDemoTest" name="autoDemoTest" time="18.142">
    <failure type="java.lang.AssertionError" message="expected [https://www.automationanywhere.com/request-live-demo-2] but found [https://www.automationanywhere.com/request-live-demo]">
      <![CDATA[java.lang.AssertionError: expected [https://www.automationanywhere.com/request-live-demo-2] but found [https://www.automationanywhere.com/request-live-demo]
at AutomationAnywhereDemoTest.autoDemoTest(AutomationAnywhereDemoTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- autoDemoTest -->
  <testcase classname="AutomationAnywhereTest" name="autoTest" time="15.894"/>
</testsuite> <!-- Surefire test -->
